search/django/weedid/views.py

```diff
@@ -30,64 +30,82 @@ def elasticsearch_query(request):
 
 def upload(request):
```

Comment:
As far as I was aware, the conventional way of requiring authentication for some views is to decorate them with `@login_required` from [`django.contrib.auth.decorators`](https://docs.djangoproject.com/en/2.0/_modules/django/contrib/auth/decorators/). So these if statements and indentation should not be necessary.